webroot/index.php

```diff
@@ -5,6 +5,13 @@
 require_once $LOC_HEADER;
 ?>
 
+<br><br>
+
+<?php
+echo $SQL->getPage($BRANDING["page"]["home"])["content"];
+?>
+
+<br>
```

Comment:
```suggestion

```
Same as above - there should be some built-in padding so this shouldn't be required.